src/zamg/zamg.py

```diff
@@ -178,9 +178,9 @@ def set_default_station(self, station_id: str):
     def last_update(self) -> datetime | None:
         """Return the timestamp of the most recent data."""
         if self._timestamp is not None:
-            return datetime.strptime(
-                self._timestamp, "%Y-%m-%dT%H:%M:%S+00:00"
-            ).replace(tzinfo=zoneinfo.ZoneInfo("Europe/Vienna"))
+            return datetime.strptime(self._timestamp, "%Y-%m-%dT%H:%M+00:00").replace(
+                tzinfo=zoneinfo.ZoneInfo("Europe/Vienna")
+            )
```

Comment:
```suggestion
            return datetime.strptime(self._timestamp, "%Y-%m-%dT%H:%Mz").astimezone(
                tz=zoneinfo.ZoneInfo("Europe/Vienna")
            )
```

@steinerl Did you mean a change like this?
Then we are getting the `last_update` correct for Austria.

Reply:
I don't see why we would need the `Europe/Vienna` timezone. The ZAMG timestamps are in `UTC` and as long as we create a datetime that includes the `UTC` timezone it should be correct.

That should be enough:
```python
return datetime.strptime(self._timestamp, "%Y-%m-%dT%H:%M%z")
```

Reply:
Yes I agree, then we should take the solution from @steinerl.
@tklecka Please change it accordingly.
And also change line 192:
`> datetime.utcnow()`
Thanks

Reply:
I would keep line 192 as is, because if you remove the `.replace(tzinfo=zoneinfo.ZoneInfo("UTC"))` part, you remove the timezone info from the datetime object.